pkg/management/component.go

```diff
@@ -33,6 +34,8 @@ const (
 
 	actionVerifyEmail       = "ct-verify-email"
 	actionVerifyPhoneNumber = "mobilephone-validation"
+
+	frontendType = "bo"
```

Comment:
Should not be defined at endpoint level ?

Reply:
Already answered for account/component.go